### DISCUSSION TITLE
Is the import/export format documented

### DISCUSSION THREAD

**Discussion by andersonjacob** [TandoorRecipes/recipes#3436] [General]
I have many recipes stored in an sqlite database of my own design.  I like Tandoor Recipes, and I'm confident that I can write a transformation from my database to the requisite json (or json + zip).  I looked around for a specification of the format, but I didn't find one.  Can anyone point me to a description of the format?  Even the import code would probably give me some hints.  

The other option I guess would be to manually input a few recipes and then do an export and try to follow that pattern.

Any thoughts?

**Comment by smilerz** [comment]
https://developers.google.com/search/docs/appearance/structured-data/recipe#recipe-properties

**Reply by andersonjacob** [reply]
That is what I was looking for.  Thank you.

**Reply by andersonjacob** [reply]
This schema doesn't seem to be the schema that is used by the export.

**Comment by andersonjacob** [comment]
I have mostly reverse engineered the export format to create some files for import, but on the import there are some errors.  What is the best way to get more information for what the import errors are.  Right now I have this very unhelpful message.

-------------------- 
ERROR 
'NoneType' object has no attribute 'keywords'
--------------------
Imported 0 recipes.
